### PR TITLE
Regenerate Knapsack report

### PR DESCRIPTION
**Pull request by sauloperez**
#### What? Why?

I don't think I've seen any update on the knapsack while I've been here and it is already asking for it:

captura de pantalla 2018-01-23 a les 8 07 57

#### What should we test?

Nothing this is a dev task only.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,7 +40,6 @@ before_script:
 
 script:
   - 'if [ "$KARMA" = "true" ]; then bundle exec rake karma:run; else echo "Skipping karma run"; fi'
-  #- "KNAPSACK_GENERATE_REPORT=true bundle exec rspec spec"
```
Comment (enricostano) [inline .travis.yml]: Can we document somewhere why and how we run this?

Comment (sauloperez) [reply .travis.yml]: Yep, we should add this somewhere in the wiki

Comment (sauloperez) [reply .travis.yml]: Done. You'll find it under the testing section of the wiki. https://github.com/openfoodfoundation/openfoodnetwork/wiki/Parallelized-test-suite

**Comment by sauloperez** [comment]
this can be merged as soon as you guys agree.

**Comment by sauloperez** [comment]
I restarted the build to see what we get. I guess we have quite a large variance.

**Comment by sauloperez** [comment]
jobs

Looks more evenly distributed this time to me. That was definitely variance @mkllnk .

**Pull request merged**